src/commands/repos/WriteRepos.ts

```diff
@@ -67,37 +67,38 @@ export class WriteRepos extends AbstractReposCommand {
                 };
                 resolve({repoName, status});
             });
-        } else if (this.freeze && this.useTags) {
+        } else if (this.useTags) {
             return Repository.getActiveTagOfReleaseBranch(repoName, this.parentRepos[repoName])
                 .then((activeTag) => {
-                    const current = this.parentRepos[repoName];
-                    const status: IRepoStatus = {
-                        url: current.url,
-                        branch: current.branch,
-                        description: current.description ? current.description : repoName
-                    };
                     if (activeTag) {
+                        const current = this.parentRepos[repoName];
+                        const status: IRepoStatus = {
+                            url: current.url,
+                            branch: current.branch,
+                            description: current.description ? current.description : repoName
+                        };
                         status.tag = activeTag;
                         status.tagMarker = activeTag;
```

Comment:
The tag and tagMarker  could also go inside the block

```suggestion
                        const current = this.parentRepos[repoName];
                        const status: IRepoStatus = {
                            url: current.url,
                            branch: current.branch,
                            description: current.description ? current.description : repoName,
                            tag: activeTag,
                            tagMarker: activeTag
                        };
```